doc/lua_api.txt

```diff
@@ -5935,9 +5937,11 @@ Player properties need to be saved manually.
     {
         hp_max = 1,
         -- For players only. Defaults to `minetest.PLAYER_MAX_HP_DEFAULT`.
+		-- Maximal value is 65535 (2^16 - 1)
 
         breath_max = 0,
         -- For players only. Defaults to `minetest.PLAYER_MAX_BREATH_DEFAULT`.
+		-- Maximal value is 65535 (2^16 - 1)
```

Comment:
These look better to me not having an extra indent, like most or all multiline comments.